penny_university_frontend/src/components/followups/FollowUpCard.js

```diff
@@ -1,12 +1,31 @@
-import React from 'react'
-import Content from '../Content'
-import FollowUpUserInfo from "./FollowUpUserInfo"
+import React, {useState} from 'react'
+import {Content, EditContent} from '../content'
+import {EditButton, SaveButton} from '../buttons'
+import FollowUpUserInfo from './FollowUpUserInfo'
+
+const FollowUpCard = ({followUp, updateFollowUp}) => {
+  const [editMode, toggleEditMode] = useState(false)
+  const [content, updateContent] = useState(followUp.content)
+
+  const saveFollowUp = () => {
+    followUp.content = content
+    updateFollowUp(followUp)
+    toggleEditMode(false)
+  }
 
-const FollowUpCard = ({followUp}) => {
   return (
     <div className='pt-2'>
-      <FollowUpUserInfo user={followUp.userInfo} date={followUp.date}/>
-      <Content className='ml-4 border-left pl-3' content={followUp.content}/>
+      <div className='d-flex justify-content-between'>
+        <FollowUpUserInfo user={followUp.userInfo} date={followUp.date}/>
+        {editMode ?
```

Comment:
Somehow this is eventually where we hook in and get whether or not this is the _authed_ user (e.g. the only one allowed to edit). Should we start creating tickets for things that will change after we have an authed user?

Reply:
At this point, it's really only editing and creating follow ups. I'm not sure if we need separate tickets for each?